### PR TITLE
Explore and bookmark page

### PR DESCRIPTION
**Pull request by VLeads**
This PR adds Homepage and explore page layout.

Temporarily  "Broadcase" button is commented.

Also a logout modal added.

👉 Just review ```Added explore-and-bookmark-page``` commit only.

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
UI looks pretty clean
Code is modular
Looks good to merge

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+
+    <!-- Rapid UI -->
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+    <link
```
Comment (Hamza-777) [inline public/index.html:30]: Font Awesome is fine but you can also try react-icons they are totally free and has a lot of variety.

**Review by Hamza-777 — APPROVED** [review]
Everything else looks good, good work.

File: src/components/sidebar/sidebar.module.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    font-size: 1.8rem;
+    width: 100%;
+}
+
+.userAccountModalOption,
+.userAccount {
+    display: flex;
+    align-items: center;
+    margin-bottom: 2rem;
+    border-radius: 50px;
```
Comment (abhisheksahani-nova) [inline src/components/sidebar/sidebar.module.css:79]: Suggestion - For border-radius 50px you can use rem

File: src/components/tweetBox/tweetBox.module.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+.tweetBox {
+    margin-top: 1.5rem;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-left: 1rem;
+}
+
+.inputBox {
+    min-height: 30px;
```
Comment (abhisheksahani-nova) [inline src/components/tweetBox/tweetBox.module.css:26]: Suggestion - For pixels higher than 10 using rem will be useful In responsiveness

**Review by abhisheksahani-nova — APPROVED** [review]
Overall everything looks good , great work with clean and readable code

**Pull request merged**